node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -342,7 +354,9 @@ impl ChainScraper {
 	where
 		Sender: overseer::DisputeCoordinatorSenderTrait,
 	{
-		let target_ancestor = get_finalized_block_number(sender).await?;
+		let target_ancestor = get_finalized_block_number(sender)
+			.await?
+			.saturating_sub(Self::SCRAPED_FINALIZED_BLOCKS_COUNT.get() - 1);
```

Comment:
what is this -1 for? If `SCRAPED_FINALIZED_BLOCKS_COUNT` is set to 1, shouldn't we be subtracting 1 instead of 0?

Reply:
You are right:
https://github.com/paritytech/polkadot/blob/1525edea8cd5bb6f47df6940481d54f80cc1871d/node/core/dispute-coordinator/src/scraping/mod.rs#L392

If we subtract 1 here we won't have 10 finalized blocks cached.